cdcpd/src/cdcpd_node.cpp

```diff
@@ -619,9 +656,13 @@ void CDCPD_Moveit_Node::reset_if_bad(CDCPD::Output const& out)
         out.status == OutputStatus::ObjectiveTooHigh)
     {
         // Recreate CDCPD from initial tracking.
-        std::unique_ptr<CDCPD> cdcpd_new(new CDCPD(nh, ph,
-            deformable_object_configuration_->initial_.points_,
-            deformable_object_configuration_->initial_.edges_,
+        bool const use_initial_state = true;
+        PointCloud::Ptr vertices =
+            deformable_object_tracking_map_.form_vertices_cloud(use_initial_state);
+        Eigen::Matrix2Xi edges =
+            deformable_object_tracking_map_.form_edges_matrix(use_initial_state);
```

Comment:
fix track map naming convention (remove underscore) as this isn't a private member